examples/python/auto-tracing/main.py

```diff
@@ -0,0 +1,8 @@
+from app.main import main
+
+import logfire
+
+logfire.configure()
+logfire.install_auto_tracing(modules=['app'])
```

Comment:
Does this work? It shouldn't.

I think this whole example would be more compelling if we also added `logfire.instrument_flask()` and `logfire.instrument_sqlalchemy()`.